public/styles.css

```diff
@@ -25,9 +25,16 @@ nav::before {
     top: 0;
     left: 0;
     background-size: cover;
-    width: 444px;
     height: 114px;
     z-index: -10;
+    width: 100%;
+    background-position-x: center;
+}
+
+@media only screen and (min-width: 445px) {
+    nav::before {
+        width: 444px;
+    }
 }
```

Comment:
Whoops, forgot I removed this when troubleshooting my last commit. This looks a little like a max-width, but 444px is actually larger than 100%.